gui/build.py

```diff
@@ -60,7 +60,13 @@ def do_run(options):
     do_uic(options)
     do_rcc(options)
     env = dict(os.environ)
-    env['PYTHONPATH'] = '.'
+    print sys.platform
+    # if runing under the OS X, we should just add '.' to
```

Comment:
the comments needs to be updated now. maybe just:
# update PYTHONPATH so python can find mozregui package